### ISSUE TITLE
Feedback for XHR requests

### ISSUE THREAD

**Issue by svenuthe** [jupyter/nbgrader#983]
### Operating system
Fedora

### `nbgrader --version`
python 3.5.3
nbgrader 0.5.4

### `jupyterhub --version` (if used with JupyterHub)
4.4.0

### `jupyter notebook --version` 
5.4.1

### Expected behaviour
Within the correction work flow, it should be obvious if an request does **not** succeed.
Option 1) Make the read background of the rating box more obvious (change  answers header bg colour?)
Option 2) Show an error when leaving the page, when there are pending requests.
Option 3) Create new User experience (far to much effort?)

### Actual behaviour
If the request succeeds, a check mark is displayed (acceptable so far).
If there is a connection or server issue and the request does not succeed, nothing happens.
The refresh button still indicates a pending request.
When I click "next", to go to next submission, the rating is lost.
No warning!

### Steps to reproduce the behavior
Use nbgrader within very crappy WLAN environments.

**Comment by jhamrick** [comment]
Yeah, that's definitely a problem that there's no warning at least. Option 2 seems reasonable.
